.github/workflows/ci.yml

```diff
@@ -5,11 +5,27 @@ env:
 jobs:
   lint:
     runs-on: ubuntu-latest
-    container: quay.io/ovirt/vdsm-test-centos-8
+    container: quay.io/alesgar/vdsm-test-centos-8
     steps:
     - uses: actions/checkout@v2
     - name: Run linters
       run: ./automation/lint.sh
+  test-storage-no-root:
+    env:
+      TRAVIS_CI: 1
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/alesgar/vdsm-test-centos-8
+      # Required to create loop devices.
+      options: --privileged
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run storage tests
+      run: |
+        ./automation/setup-storage.sh
+        chown -R vdsm:kvm ./ /var/tmp/vdsm*
+        su -c "tox -e storage -- --cov-fail-under 60 \
+           -m 'not (integration or slow or stress or root)' storage" -- vdsm
```

Comment:
I think we need:
- tests-storage
- tetsts-storage-root

Both can use the same tests-storage.sh script, with additional argument so the script can run tests as root or as regular user.

Reply:
Was a test for running with vdsm user, I will remove the change before merging. I have it covered in a different branch and is as you suggest, two jobs for storage testing.